examples/pytorch/02-start-with-dask.ipynb

```diff
@@ -73,27 +71,29 @@
     "\n",
     "results = [x.result() for x in futures_computed]\n",
     "results"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "You should see a result of `[1, 81, 15625, 3486784401, 3138428376721]`.\n",
     "\n",
     "This was _somewhat_ of a toy example--you probably want to do more complex computations than exponents with Dask. However the core concept of making a function and then running it in a distributed fashion is at the core of what you can do with Dask on Saturn Cloud.\n",
     "\n",
     "When you're done, you can close the connection to the cluster:"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "client.close()"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
```

Comment:
I'm a little wary of this reordering but I don't think it should impact things. I'm going to kick of integration tests just in case.

Reply:
https://github.com/saturncloud/release-images/runs/3205501538?check_suite_focus=true

Reply:
I don't know why that got switched in this PR... maybe because I branched from an older version of main or something?

Reply:
I think it's something your editor did, but seems to run fine.